target-design/switch/baseport.h

```diff
@@ -45,6 +53,28 @@ BasePort::BasePort(int portNo, bool throttle)
 {
 }
 
+int BasePort::push_input(switchpacket *sp)
+{
+    int ethtype, ctrl, quanta;
+
+    // Packets smaller than three flits are too small to be valid
+    if (sp->amtwritten < 3)
+        return 0;
```

Comment:
seems like this is conflating errors with logging? should we just abort in this case?